src/test/java/com/snowflake/kafka/connector/internal/streaming/SnowflakeSinkServiceV2IT.java

```diff
@@ -770,126 +770,6 @@ public void testMissingPropertiesForStreamingClient() {
     }
   }
 
-  /* Service start -> Insert -> Close. service start -> fetch the offsetToken, compare and ingest check data */
-
-  @Test
-  public void testStreamingIngestionWithExactlyOnceSemanticsNoOverlappingOffsets()
```

Comment:
this is a streaming ingest test!

Reply:
should we keep some of the e2e tests as well? or are those situations redundant with streaming's tests?

Reply:
Snowpipe + Exactly once is totally independent from snowpipe streaming. 
There are exactly once tests for snowpipe streaming because we support that. 

We will no longer support snowpipe with exactly once. 

End to end tests which I was suggesting earlier was all ingestionmethod = snowpipe and delivery guarantee = exactly once

We still want to keep snowpipe streaming tests with exactly once (there is no atleast once in snowpipe streaming)

Reply:
sweet, thanks for clearing that up